tests/lib/prepare.sh

```diff
@@ -379,11 +383,13 @@ EOF
     # on core18 we need to use the modified snapd snap and on core16
     # it is the modified core that contains our freshly build snapd
     if is_core18_system; then
-        extra_snap="$IMAGE_HOME"/snapd_*.snap
+        extra_snap=("$IMAGE_HOME"/snapd_*.snap)
     else
-        extra_snap="$IMAGE_HOME"/core_*.snap
+        extra_snap=("$IMAGE_HOME"/core_*.snap)
     fi
 
+    # shellcheck disable=SC2128
+    # extra_snap should contain only ONE snap
```

Comment:
This is https://github.com/koalaman/shellcheck/wiki/SC2128

Could we expand the first element ourselves?

Reply:
I need to doble check with bash. IIRC in zsh, if `var` ended up being a single string, expanding `${var[0]}` would yield the first letter of that string.

Reply:
BTW. ${var} if var is an array gives the first element.

Reply:
Let's do that. I'm mainly after not having shellcheck exceptions. If you choose to do this please add a comment, I'm not sure if the particular array semantics is widely known.